specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-04-01/registeredidentities.json

```diff
@@ -0,0 +1,372 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-04-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/certificates/{certificateName}": {
+      "put": {
```

Comment:
>put [](http://example.com/codeflow?start=7&length=3)

you are missing get, delete calls, see: [Inconsistent or incomplete CRUD](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review) from our documentation for more information.

Reply:
Sorry as I was just adding the new api version, I just copied the existing json files without any changes. My changes are mainly in the vaults.json and not in the other 3 json swaggers

Reply:
I will still discuss this with the team but can this be taken later as this is the existing swagger and this change is outside the scope of the current changes?

Reply:
Got it, thanks for the clarification, will focus on the updates only.